packages/esm-patient-notes-app/src/notes/visit-notes.resource.ts

```diff
@@ -103,9 +104,9 @@ export function useInfiniteVisits(patientUuid: string) {
   };
 }
 
-export function fetchConceptDiagnosisByName(searchTerm: string) {
+export function fetchConceptDiagnosisByName(searchTerm: string, diagnosisConceptClass: string) {
   return openmrsObservableFetch<Array<Concept>>(
-    `${restBaseUrl}/concept?q=${searchTerm}&searchType=fuzzy&class=8d4918b0-c2cc-11de-8d13-0010c6dffd0f&q=&v=custom:(uuid,display)`,
+    `${restBaseUrl}/concept?name=${searchTerm}&searchType=fuzzy&class=${diagnosisConceptClass}&name=&v=custom:(uuid,display)`,
```

Comment:
@nanfuka Is the `name` param intentionally passed twice in the above url?

Reply:
Whoops. Nice catch, Dan. I've pushed a commit that fixes this.